portfolio/src/main/webapp/script.js

```diff
@@ -84,13 +93,15 @@ function getRandomQuoteUsingArrowFunctions() {
 async function getRandomQuoteUsingAsyncAwait() {
   const response = await fetch('/data');
   const quote = await response.text();
-  document.getElementById('quote-container').innerText = quote;
+  document.getElementById('comment_author').innerText = comment_author;
+  document.getElementById('text-input').innerText = comment;
+  //document.getElementById('quote-container').innerText = quote;
 }
 
 /**
  * Fetches quotess from the servers and adds them to the DOM.
  */
-function getQuote() {
+/*function getQuotesFromServer() {
```

Comment:
I saw that you call "getQuotesFromServer()" in index.html, but this function is commented out? Nonetheless I still think you should make this change in PR for part 3 instead.

Reply:
I think what happened is that I had originally made changes and tried pushing to the Wk2Prt3Anna branch and there was an error because I think I hadn't resolved your comment on it (not really sure), but before I realized that I switched to this branch and I think it got rid of the changes I had made that I planned to push to Wk2Prt3Anna?

Reply:
You can switch back to the Wk2Prt3Anna branch and made the changes there, see my other comment in index.html for this. Let me know if you have any issues doing that